dev/TreeView/TreeViewNode.cpp

```diff
@@ -284,6 +284,8 @@ hstring TreeViewNode::GetContentAsString()
         {
             return result.ToString();
         }
+
+        return unbox_value<winrt::hstring>(content);
```

Comment:
This will crash if the content is not a string. Perhaps try_unbox_value ?

Reply:
I remember there was a try_unbox_value function, but looks like it's gone? I found `unbox_value_or`, which seems to do similar things.

Reply:
Ah yeah, that sounds right. Thanks!